.github/workflows/luacheck.yml

```
@@ -0,0 +1,13 @@
+name: luacheck
+on: [push, pull_request]
+jobs:
+  build:
```

Suggestion to use better name for luacheck job, maybe luacheck or lint. This is different from "name: luacheck" and is used to distinguish different workflow jobs for GitHub task selection in checks and such.
```suggestion
  luacheck:
```